ROMFS/px4fmu_common/init.d/airframes/13015_jiyi_batman

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+#
+# @name JIYIRobot Batman
+#
+# @type VTOL Tiltrotor
+# @class VTOL
+#
+# @maintainer Stone White <stone@thone.io>
+#
+# @output MAIN1 Motor right
+# @output MAIN2 Motor left
+# @output MAIN3 Motor back
+# @output MAIN4 empty
+# @output MAIN5 Tilt servo right
+# @output MAIN6 Tilt servo left
+# @output MAIN7 Elevon right
+# @output MAIN8 Elevon left
+#
+# @board intel_aerofc-v1 exclude
+# @board bitcraze_crazyflie exclude
+#
```

Comment:
If you don't plan to enable the usage of a FMU-v2 on this vehicle, add an exclude for it would solve the flash issues there. See https://github.com/PX4/Firmware/blob/cfbf69f2a7c408903864300bd24cd3939258a6c1/ROMFS/px4fmu_common/init.d/airframes/13014_vtol_babyshark#L19.

Reply:
excluded